zookeeper-command-line-client/pom.xml

```diff
@@ -70,6 +70,15 @@
   </dependencies>
   <build>
     <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <configuration>
+          <compilerArgs>
+            <arg>-Xlint:all</arg>
```

Comment:
Is this to switch off `-Werror`?

Reply:
I suggest to add this dependency to the pom instead (see also comment in the parent pom for this dep).
```
    <dependency>
      <!-- Needed by zookeeper, which only has an optional dependency. -->
      <groupId>com.github.spotbugs</groupId>
      <artifactId>spotbugs-annotations</artifactId>
    </dependency>
```

Reply:
Thanks! Will use this